controllers/topology/internal/log/log.go

```diff
@@ -115,6 +115,12 @@ func (l *topologyReconcileLogger) V(level int) Logger {
 // Infof logs to the INFO log.
 // Arguments are handled in the manner of fmt.Printf.
 func (l *topologyReconcileLogger) Infof(msg string, a ...interface{}) {
+	// If the logger implements CallDepthLogger, let's use WithCallDepth
```

Comment:
We could also do this in `LoggerFrom` above (but we have to pick up at lesat klog v2.20.0) because in earlier version the callDepth is resetted when `V` or any of the `With` funcs is called.

Reply:
Should we log an issue to revisit this as soon as we upgrade to keep track of this?

Reply:
Yup let's do that.

Reply:
I just mentioned it here instead: https://github.com/kubernetes-sigs/cluster-api/issues/5576